test/DynamoCoreTests/Updates/UpdatesTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.IO;
+using Dynamo.Updates;
+using NUnit.Framework;
+
+namespace Dynamo.Tests
+{
+    /// <summary>
+    /// Test class to test the UpdateManager class and its related classes
+    /// </summary>
+    [TestFixture]
+    public class UpdatesTests
+    {
+        private UpdateManager updateManager;
+
+        [SetUp]
+        public void Init()
+        {
+            var config = UpdateManagerConfiguration.GetSettings(null);
```

Comment:
@QilongTang won't this make a network request to s3? What about setting disable on the config.

Reply:
@mjkkirschner I thought the default behavior now is disabled since we are no longer checking newer builds, let me double check..
```
            DownloadSourcePath = PRODUCTION_SOURCE_PATH_S;
            SignatureSourcePath = PRODUCTION_SIG_SOURCE_PATH_S;
            CheckNewerDailyBuild = false;
            ForceUpdate = false;
            InstallerNameBase = INSTALL_NAME_BASE;
            DisableUpdates = false;
```

Reply:
@mjkkirschner Not even related to default behavior, since lookup is null, there is no source to ping at all here